temba/channels/types/telegram/type.py

```diff
@@ -46,11 +47,37 @@ def deactivate(self, channel):
         bot = telegram.Bot(config['auth_token'])
         bot.delete_webhook()
 
+    def get_quick_replies(self, metadata, post_body):
+        metadata = json.loads(metadata)
+        quick_replies = metadata.get('quick_replies', None)
+        url_buttons = metadata.get('url_buttons', None)
+        replies = []
+        keyboard_type = None
+
+        if quick_replies:
+            for reply in quick_replies:
+                replies.append([dict(text=reply.get('title'), callback_data=reply.get('payload'))])
+            keyboard_type = 'keyboard'
+        elif url_buttons:
```

Comment:
URL buttons are accepted by all channels that support quick replies, so we think that could implement them as well, and a good example for that, would be when the flow of the U-Report shares a link to poll results with his users, in this case, could use an URL button.

![21015445_10211976244660395_2029988703_n](https://user-images.githubusercontent.com/5198870/29533983-986df988-868a-11e7-9229-217b1a7da7c6.jpg)